packages/big-design/src/components/FileUploader/types.ts

```diff
@@ -11,3 +13,7 @@ export interface ValidatorConfig {
   type: string;
   validator: (file: File) => boolean;
 }
+
+export interface Action extends Omit<ButtonProps, 'variant'> {
```

Comment:
should we `omit` `icons` and `margin` props?

Reply:
Updated

```
export interface Action
  extends Omit<
    ButtonProps,
    | 'variant'
    | 'children'
    | 'iconOnly'
    | 'iconRight'
    | 'iconLeft'
    | 'margin'
    | 'marginTop'
    | 'marginRight'
    | 'marginBottom'
    | 'marginLeft'
    | 'marginVertical'
    | 'marginHorizontal'
  > {
  label: string;
}
```